sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Trigger.java

```diff
@@ -30,7 +30,7 @@
 import org.joda.time.Instant;
 
 /**
- * {@link Trigger Triggers} control when the elements for a specific key and window are output. As
+ * Triggers control when the elements for a specific key and window are output. As
```

Comment:
You just don't like the self-link? My default is to link the heck out of things - you never know when someone might want to click or copy link destination, etc.

Reply:
I've gotten various feedback (both from javadoc and from text such as the programming guide) that a) having `Object`s where the s is hanging out separately is strange looking/undesirable, and b) having `Objects` all in code font when the object is actually just named `Object` is incorrect. in this case these instances seemed very generic and talking about triggers as nouns like many other places in this file's javadoc so it didn't seem to lose much to have them as words. I see your point about links though.

Is there an official javadoc specific style guide? since we don't specify a preferred website style guide for Beam content, I was following [this public one](https://developers.google.com/style/code-in-text#keywords) which recommends option c) adding "object" or "instance" if you need to make something plural rather than a) or b) options above. So it'd look like:

   Several predefined {@link Trigger} objects are provided:

all of that said, I know that this issue exists throughout our javadoc and this is more of a "whack-a-mole as people report them" approach that might not be worth it. Thoughts?

Reply:
you know, I'm just going to revert this file -- this is a broader consistency issue that needs a better plan. the Coder change is worthwhile I think so I'll keep that one.